data.sql

```diff
@@ -122,15 +132,69 @@ INSERT INTO animals (
   )
 VALUES(11, 'Ditto', '2022-05-14', 4, TRUE, 22, null);
 
-INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melod
-y Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
-INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melod
+y Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+INSERT INTO species (name)
+VALUES ('Pokemon'),
+  ('Digimon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 1
+WHERE name NOT LIKE '%mon';
+
+UPDATE animals
+SET owner_id = 1
+WHERE name = 'Agumon';
+
+UPDATE animals
+SET owner_id = 2
+WHERE name = 'Gabumon'
+  OR name = 'Pikachu';
+
+UPDATE animals
+SET owner_id = 3
+WHERE name = 'Devimon'
+  OR name = 'Plantmon';
+
+UPDATE animals
+SET owner_id = 4
+WHERE name = 'Charmander'
+  OR name = 'Squirtle'
+  OR name = 'Blossom';
 
-UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
-UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+UPDATE animals
+SET owner_id = 5
+WHERE name = 'Angemon'
+  OR name = 'Boarmon';
 
```

Comment:
Please Insert the following data for specializations:
`   
 Vet William Tatcher is specialized in Pokemon.
 Vet Stephanie Mendez is specialized in Digimon and Pokemon.
 Vet Jack Harkness is specialized in
`

Reply:
Thanks for pointing out this part of the project. I completely forgot to add queries to my data.sql file.